Dockerfile

```diff
@@ -8,8 +8,10 @@ RUN apt-get update && apt-get install -y \
     wget \
     python3 \
     python3-pip \
+    && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
 RUN pip3 install gpustack
+ENV LD_LIBRARY_PATH=/usr/local/cuda-12.5/compat:${LD_LIBRARY_PATH}
```

Comment:
Since the directory will change depending on the passed CUDA_VERSION, locking to `cuda-12-5` may cause problems for other versions.
Can it work properly in your environment if you set it to use the current default CUDA version like below?
`ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:/usr/local/cuda/compat:${LD_LIBRARY_PATH}`